tests/storage-vm

```diff
@@ -301,6 +301,16 @@ for poolDriver in $poolDriverList; do
         lxc start v1
         waitInstanceReady v1
 
+        echo "==> Checking /dev/disk/by-id representations are correct"
+        lxc exec v1 -- stat -c "%F" /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_lxd_block1ro | grep -xF "symbolic link"
+	lxc exec v1 -- readlink -f /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_lxd_block1ro | grep -xF "/dev/sdb"
```

Comment:
IIRC, we want to use `/dev/disk/by-id` to avoid depending on automatically allocated `/dev/sdX` style as the later are not guarantied to be stable.

Reply:
I figured guaranteeing the link in properly set up took care of the problem, would it be better to just substitute every `/dev/sd*` reference?